From dfad7a3f389c3b852f45563854df123ac6facdec Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Mon, 15 Jul 2019 00:09:18 +0200
Subject: Add missing QVector include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Summary:
My build was failing with
```
In file included from /home/nico/kde/src/ark/kerfuffle/pluginmanager.cpp:28:
/home/nico/kde/src/ark/kerfuffle/pluginmanager.h:138:22: error: field ‘m_plugins’ has incomplete type ‘QVector<Kerfuffle::Plugin*>’
     QVector<Plugin*> m_plugins;
                      ^~~~~~~~~
In file included from /usr/include/qt/QtCore/qglobal.h:1204,
                 from /usr/include/qt/QtCore/qnamespace.h:43,
                 from /usr/include/qt/QtCore/qobjectdefs.h:48,
                 from /usr/include/qt/QtCore/qobject.h:46,
                 from /usr/include/qt/QtCore/QObject:1,
                 from /home/nico/kde/src/ark/kerfuffle/plugin.h:33,
                 from /home/nico/kde/src/ark/kerfuffle/pluginmanager.h:31,
                 from /home/nico/kde/src/ark/kerfuffle/pluginmanager.cpp:28
```

Test Plan: Builds. Can open a file

Reviewers: #ark, elvisangelaccio

Reviewed By: #ark, elvisangelaccio

Subscribers: kde-utils-devel, #ark

Tags: #ark

Differential Revision: https://phabricator.kde.org/D22461
---
 kerfuffle/pluginmanager.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kerfuffle/pluginmanager.h b/kerfuffle/pluginmanager.h
index 75a971b..8b9ad69 100644
--- a/kerfuffle/pluginmanager.h
+++ b/kerfuffle/pluginmanager.h
@@ -31,6 +31,7 @@
 #include "plugin.h"
 
 #include <QMimeType>
+#include <QVector>
 
 namespace Kerfuffle
 {
-- 
cgit v1.1
